managers/bosh-manager/DirectorService.js

```diff
@@ -152,6 +190,11 @@ class DirectorService extends BaseDirectorService {
     }
   }
 
+  isRestorePossible(plan_id) {
```

Comment:
Same here as well...we could get all from plan and make it a util function instead of having in DirectorService.

Reply:
done